Src/ReleaseHistory.md

```diff
@@ -16,6 +16,10 @@
 - UER => eliminate unhandled exceptions in rules
 - UEE => eliminate unhandled exceptions in engine
 
+## Unreleased
+
+- Bump MongoDB.Driver from 2.13.1 to 2.15.0 and Microsoft.AspNetCore.Http from 2.1.0 to 2.2.0. [#608](https://github.com/microsoft/sarif-pattern-matcher/pull/608)
```

Comment:
>-  [](http://example.com/codeflow?start=0&length=2)

Do we existing tests in both the security project and spam function that exercise a code path that will leverage these dependencies?  i.e. will we know if somehow we break ourselves due to any binding or other runtime behaviors.

Reply:
for the mongodb package, we have an end-to-end test
for the http package, its the azure function. I was able to run and debug.

Reply:
actually, our mongodb validator is disabled due to not being signed. so, we won't face any impact.